Application/src/view/visuals/Draw.java

```diff
@@ -13,9 +13,9 @@
 import java.util.ArrayList;
 
 /**
- * @version 2.0
+ * @version 2.1
  * @author Pontus Laestadius, Sebastian Fransson
- * Collaborator Rashad Kamsheh, Kosara Golemshinska, Isabelle Törnqvist
+ * Collaborators Rashad Kamsheh, Kosara Golemshinska, Isabelle Törnqvist
```

Comment:
Add your name maybe?

Reply:
I dont feel like 2 lines of code is enough for that

Reply:
fair